modules/check_in/config/locales/exceptions.en.yml

```diff
@@ -26,6 +26,18 @@ en:
         code: 'LOROTA-API_401'
         detail: 'Authentication Error'
         status: 401
+      CIE-VETS-API_404:
```

Comment:
I'm not sure if this is needed. This is used to map downstream errors through a middleware layer, but in this case we're not using it. We're instead raising a custom exception(s) in these scenarios.

Reply:
Based on the research for throwing a custom exception, the backend server seems to be using `exceptions.en.yml` to map exceptions thrown from the service and define the status/code sent to the front end.

[backend_service_exception/validate_arguments](https://github.com/department-of-veterans-affairs/vets-api/blob/010ade1f5de95b9bdcb9bb2682416428cd03109f/lib/common/exceptions/backend_service_exception.rb#L94-L97) checks for the key & throws error if not defined.  Pls let me know we can sync up when you have sometime

Reply:
you're right, thanks for clarifying.